util/storage-samples/storage-di-sample/src/main/java/org/fidoalliance/fdo/storage/DiDbStorage.java

```diff
@@ -154,9 +154,19 @@ public Composite createVoucher(Object createParams) {
                         this.onDieService.getOnDieCache());
 
         // validate test signature against certpath
+
+        // create proper signed data format
+        // Note: protected header in second_key is empty since
+        // we have no nonce with the test signature.
+        Composite sigStruct = Composite.newArray()
+                .set(Const.FIRST_KEY, "Signature1")
+                .set(Const.SECOND_KEY, new byte[0])
```

Comment:
Doesn't COSE require at least algorithm_ID in the protected headers?

Reply:
Yes, it should be there. I've now updated the DAL devices to include it as well.